### PR TITLE
ETL: Add section about Apache Hop

### PR DESCRIPTION
**Pull request by amotl**
## About
@wierdvanderhaar and @hlcianfagna recently verified connectivity to Apache Hop, aka. the »Hop Orchestration Platform«, thanks! This patch intends to add a corresponding item to the documentation.

## Preview
https://crate-clients-tools--64.org.readthedocs.build/en/64/integrate/etl.html#apache-hop

## References
- https://hop.apache.org/
- https://github.com/apache/hop/

### CODE REVIEW

File: docs/integrate/etl.md [hunk 1]
```diff
@@ -104,6 +104,32 @@ A few companies are specializing in offering managed Flink services.
...
+- **Workflows and pipelines** can be designed in the Hop Gui and run on the Hop
+  native engine (local or remote), or on Spark, Flink, Google Dataflow or AWS EMR
+  through Beam. _Design once, run anywhere._
+
+- **Lifecycle Management** enables developers and administrators to switch between
+  projects, environments and purposes without leaving your train of thought.
+
+```{seealso}
+[CrateDB and Apache Hop]
+```
```
Comment (amotl) [inline docs/integrate/etl.md:130]: There are no references to relevant tutorials yet. Just let us know if you intend to publish a corresponding article on the community forum, whether you would want us to do it, extended by exercising a relevant snippet on behalf of [cratedb-examples](https://github.com/crate/cratedb-examples), or if you can do that.

A very little walkthrough in form of a README.md, added on behalf of a patch and PR, will be sufficient to enable others to continue working from there.

In any case, it is not a blocker for this to be merged.

/cc @matkuliak

Comment (hlcianfagna) [reply docs/integrate/etl.md:141]: We are reviewing a draft for a tutorial, please wait to merge this until the tutorial is published as there are some details to consider to make it work with CrateDB

Comment (hlcianfagna) [reply docs/integrate/etl.md:141]: Apologies for the delay, the tutorial is out now https://community.cratedb.com/t/using-apache-hop-with-cratedb/1754

Comment (hlcianfagna) [reply docs/integrate/etl.md:141]: Yes, but could we link the tutorial? because it does not work out-of-the-box without the workaround for `pg_user`

Comment (amotl) [reply docs/integrate/etl.md:132]: Sure. The tutorial is linked into the CrateDB Guide.

-- https://github.com/crate/cratedb-guide/pull/83

**Pull request merged**